### ISSUE TITLE
Contents example with packager result in conflict

### ISSUE THREAD

**Issue by PierreF** [goreleaser/nfpm#302]
I'm trying to use [example](https://nfpm.goreleaser.com/configuration/) to include different file per-packager (rpm, deb...), 
but it result in "content collision".

Minimal nfpm.conf to reproduce:
```
name: "foo"
# This is a copy/paste from nfpm documentation
contents:
  - src: path/to/rpm/file.conf
    dst: /etc/file.conf
    type: config|noreplace
    packager: rpm
  - src: path/to/deb/file.conf
    dst: /etc/file.conf
    type: config|noreplace
    packager: deb
```

Step to produce the error:
```
mkdir -p path/to/rpm/ path/to/deb/
touch path/to/rpm/file.conf path/to/deb/file.conf

nfpm pkg --packager deb --target /tmp/
```

Error:
```
nfpm: error: cannot add path/to/deb/file.conf because path/to/rpm/file.conf is already present at the same destination (/etc/file.conf): content collision
```

**Comment by erikgeiser** [comment]
I guess that's my bad. I introduced the collision protection without realizing that you can now specify the packager on a per-file basis.

**Comment by erikgeiser** [comment]
Should be fixed by #303.
